index.bs

```diff
@@ -587,6 +587,44 @@ See also:
 * [[#secure-context]]
 * [[#consent]]
 
+<h3 id=text-formats>Make textual formats for people</h3>
+
+Design textual formats to be both produced and consumed by people.
+
+People who are presented with a textual format will
+expect to be able to use a text editor
+to produce or modify content.
+People who edit text will introduce a range of errors, but
+could struggle to identify and fix those errors.
+
+People expect some amount of flexibility in terms of how their edits are processed.
+Allowing flexibility in whitespace, quoting, delimiters, and other syntactic elements ensures
+that content is still comprehensible.
```

Comment:
This is not about comprehensibility, but ease of editing / error prevention.

Might be good to link to the [robustness principle](https://en.wikipedia.org/wiki/Robustness_principle) somewhere.

Reply:
I literally wrote an RFC that says that the robustness principle harms interoperability: https://www.rfc-editor.org/rfc/rfc9413.html

I have friends who like to troll me with the phrase "Postel was wrong", but I'm guessing that didn't know :)

Either way, I think that this would be a mistake for the reasons stated in that document (see [this bit](https://www.rfc-editor.org/rfc/rfc9413#section-2.3-3)).  I think that your point about this being an example is a good one though.  I've suggested something below.